analytics-core/src/main/java/com/liferay/analytics/android/model/IdentityContext.kt

```diff
@@ -14,28 +14,32 @@
 
 package com.liferay.analytics.android.model
 
-import java.util.Locale
-import java.util.UUID
+import org.koin.standalone.KoinComponent
+import org.koin.standalone.inject
+import java.util.*
 
 /**
  * @author Igor Matos
  * @author Allan Melo
  */
-internal data class IdentityContext(var dataSourceId: String) {
+internal data class IdentityContext(var dataSourceId: String): KoinComponent {
+	@delegate:Transient private val analyticsContext: AnalyticsContext by inject()
 
 	var identity: Identity? = null
 	val language: String = Locale.getDefault().toString()
 	val platform = "Android"
-	var timezone: String? = null
+	val timezone: String = TimeZone.getDefault().displayName
```

Comment:
@victorlaerte, just to double check on timezone conversion. Are we sending the event dates in UTC?

Reply:
Good catch @marcellustavares, despite the fact that I've changed the date format in JSONParser to use ISO8601 standard for UTC timezone when I double checked this right now I discovered that the default behavior for Gson is to use the local time.
I'll create a custom Type Adapter for dates and fix it.
Thanks.
https://github.com/google/gson/issues/281

Reply:
Done